pkg/plugin/keyvault_test.go

```diff
@@ -99,27 +97,27 @@ func TestNewKeyVaultClient(t *testing.T) {
 		},
 		{
 			desc:             "no error with double quotes",
-			config:           &config.AzureConfig{ClientID: "clientid", ClientSecret: "clientsecret"},
+			config:           &config.AzureConfig{ClientID: "clientid", ClientSecret: "clientsecret", TenantID: "tenantId"},
 			vaultName:        "\"testkv\"",
 			keyName:          "\"key1\"",
 			keyVersion:       "\"262067a9e8ba401aa8a746c5f1a7e147\"",
 			proxyMode:        false,
 			expectedVaultURL: "https://testkv.vault.azure.net/",
 		},
-		{
-			desc:             "no error with proxy mode",
-			config:           &config.AzureConfig{ClientID: "clientid", ClientSecret: "clientsecret"},
-			vaultName:        "testkv",
-			keyName:          "key1",
-			keyVersion:       "262067a9e8ba401aa8a746c5f1a7e147",
-			proxyMode:        true,
-			proxyAddress:     "localhost",
-			proxyPort:        7788,
-			expectedVaultURL: "http://localhost:7788/testkv.vault.azure.net/",
-		},
+		//{
```

Comment:
TODO

Reply:
Tactically: we will remove HTTP proxy support, and the user of this will migrate, as proxying Entra traffic through plaintext is not what we want anyway.